cluster-scope/overlays/ocp-staging/apiservers/cluster.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: config.openshift.io/v1
+kind: APIServer
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    oauth-apiserver.openshift.io/secure-token-storage: "true"
+    release.openshift.io/create-only: "true"
+  name: cluster
+spec:
+  audit:
+    profile: Default
+  servingCerts:
+    namedCertificates:
+    - names:
+      - "api.ocp-staging.massopen.cloud"
+      servingCertificate:
+        name: "api-certificate-letsencrypt"
+    - names:
```

Comment:
Do we need the entry for `*.apps` here? It wasn't required for ocp-prod (see 01e0f9aa).

Reply:
yes, my bad. the API server isn't serving on `*.apps`